src/packages/cli/package.json

```diff
@@ -70,8 +70,6 @@
     "@prisma/sdk": "workspace:*",
     "@prisma/studio": "0.245.0",
```

Comment:
Ideally, I'd like to get rid of @prisma/studio as well, but it seems studio-server does not ship with the static assets (JS bundles / images etc) required to show things in the browser. I will change studio-server so that it does, and will send you a PR removing this.